pkg/apis/openebs/v1alpha1/blockdevice_types.go

```diff
@@ -123,9 +123,24 @@ type DeviceCapacity struct {
 
 // DeviceDetails represent certain hardware/static attributes of the block device
 type DeviceDetails struct {
-	// DeviceType represents the type of drive like SSD, HDD etc.,
+	// DeviceType represents the type of device like
+	// sparse, disk, partition, lvm, raid
```

Comment:
Which probe populate this value?
Is there any chance we can get an empty value of it?

Reply:
it can be filled in either by udevprobe or sysfs probe. No there will be no empty value in `deviceType`.